src/store/modules/auth.js

```diff
@@ -94,69 +173,134 @@ const actions= {
-            }
-            commit('setUser', form.email, token )
-        })
-        .catch(err => {
-            console.log(err)
-            localStorage.removeItem('token')
-        })
-        // const login = new FetchAPI();
-        // // form = login.authUser('/login', form)
-        // await commit('setUser', form.get('name'))
-        
+        //const res = await axios.post('/identity/login', form)
+        const login = new FetchAPI();
+        const res = await login.authUser(process.env.VUE_APP_login, form);
+        try {
+          commit('addMessage', res.data.error_messages);
+          commit('addData', res.data.status);
+
+          const token = res.data.data.token;
+          if(res.data.status === 'success'){
+              localStorage.setItem('token',token);
+          }
+        } catch (error) {
+          console.log(error)
+        }  
     },
      
     async Verification({commit},verify){
-        await axios.post('/identity/verify',verify)
-        .then(response=>{
-            commit('addveryError',response.data.error_messages);
-            commit('addverySuccess',response.data.status);
-
-        }).catch(err => console.log(err))
+        //const res = await axios.post('/identity/verify',verify);
+        const varify = new FetchAPI();
+        const res = await varify.authUser(process.env.VUE_APP_verify,verify)
+        try {
+          commit('addveryError',res.data.error_messages);
+          commit('addverySuccess',res.data.status);
+        } catch (error) {
+          console.log(error);
+        }
 
     },
+    async fbLogin({commit},token){
+      try {
+        const fbLogIn = new FetchAPI();
+        const res = await fbLogIn.authUser(process.env.VUE_APP_fbLogin,{token:token});
+
+        localStorage.setItem('token',res.data.data.token);
       
+        //const token = localStorage.setItem('token',res.token)
+      } catch (error) {
+        console.log(error);
+      }
+    },
+
+    // load facebook SDK for login
+    loadFacebookSDK() {
+      var js,
+        fjs = document.getElementsByTagName("script")[0];
+      if (document.getElementById("facebook-jssdk")) {
+        return;
+      }
+      js = document.createElement("script");
+      js.id = "facebook-jssdk";
+      js.src = "https://connect.facebook.net/en_US/sdk.js";
+      fjs.parentNode.insertBefore(js, fjs);
+    },
+
+    // Initiate facebaook App
+    initFacebook() {
+      window.fbAsyncInit = function() {
+        window.FB.init({
+          appId:"677190916279332", //You will need to change this
+          cookie: true, // This is important, it's not enabled by default
+          version: "v13.0"
+        });
+      };
+    },
+
     async LogOut({commit}){
         let user = null;
         localStorage.removeItem('token');
+        localStorage.removeItem('userData');
+        localStorage.removeItem('vuex');
+        localStorage.removeItem('userId');
+        localStorage.removeItem('role');
```

Comment:
All of this should be maintained in State not Localstorage, I think only the auth token should be in Storage

Reply:
Resolved